### ISSUE TITLE
qm container: force use iptables-nft and avoid iptables-legacy

### ISSUE THREAD

**Issue by dougsland** [containers/qm#267]
RHEL kernel only support iptables-nft, we need to make sure in the container you also have iptables-nft installed
```
bash-5.1# rpm -qa | grep -i iptables
iptables-legacy-libs-1.8.8-6.el9.2.x86_64
iptables-libs-1.8.8-6.el9.x86_64
iptables-legacy-1.8.8-6.el9.2.x86_64
```

avoid such error when creating a container with podman:
```
Error: netavark: code: 3, msg: modprobe: FATAL: Module ip_tables not found in directory /lib/modules/5.14.0-370.331.el9iv.aarch64
iptables v1.8.8 (legacy): can't initialize iptables table `nat': Table does not exist (do you need to insmod?)
Perhaps iptables or your kernel needs to be upgraded.
```

**Comment by Yarboa** [comment]
@dougsland 
Isnt it resolved by this? #268

**Comment by dougsland** [comment]
@Yarboa yes, thanks for the heads up! 
Resolved!
